packages/frontend/src/lib/QuestionsEditor.svelte

```diff
@@ -13,19 +13,17 @@
 	const dispatch = createEventDispatcher();
 	let indexQ : HTMLDivElement;
 
-	function moveDown() {
-		let currentIndex = parseInt(indexQ.getAttribute('id')); // currently unsure of how to retrieve id
-		indexQ.setAttribute('id', (currentIndex + 1).toString());
-		let temp = questions[currentIndex];
-		questions[currentIndex] = questions[currentIndex + 1];
-		questions[currentIndex + 1] = temp;
+	function getIndex() {
+		let index : string = indexQ.getAttribute('id')!;
+		return parseInt(index);
 	}
-	function moveUp(){
-		let currentIndex = parseInt(indexQ.getAttribute('id')); // currently unsure of how to retrieve id
-		indexQ.setAttribute('id', (currentIndex - 1).toString());
+
+	function swapIndex(increment : number) {
```

Comment:
This needs to be something like
```suggestion
	function move(oldIndex: number, newIndex: number) {
```